spec/controllers/forms_controller_spec.rb

```diff
@@ -94,6 +107,19 @@
         end
       end
     end
+    context 'current user is nil' do
```

Comment:
It'd be nice if these tests also caught people mixing up the fields. That is, perhaps they could not all be bananas.